post-merge updates: beacon chain page (#7266)
ethereum/ethereum-org-website: src/content/upgrades/beacon-chain/index.md

image: ../../../assets/upgrades/core.png
summaryPoint1: The Beacon Chain doesn't change anything about the Ethereum we use today.
summaryPoint2: It introduced proof-of-stake to the Ethereum ecosystem.
summaryPoint3: It was merged with the original Ethereum proof-of-work chain in 2022.

> summaryPoint4: It is now handling all consensus logic and block gossip for Ethereum.

Reviewer: We don't need to refer to how things used to happen here.

```suggestion
summaryPoint4: It handles all consensus logic and block gossip for Ethereum.
```

---

<UpgradeStatus isShipped dateKey="page-upgrades-beacon-date">

>   The Beacon Chain shipped on December 1, 2020. At the time it was a separate chain to Ethereum Mainnet. It ran proof-of-stake but did not handle any transactions. The Beacon Chain was merged with the original Ethereum Mainnet in 2022 to take over the consensus logic and block propagation for Ethereum. Since The Merge, there has only been one, single Ethereum chain.

Reviewer: comma

```suggestion
  The Beacon Chain shipped on December 1, 2020. At the time, it was a separate chain to Ethereum Mainnet. It ran proof-of-stake but did not handle any transactions. The Beacon Chain was merged with the original Ethereum Mainnet in 2022 to take over the consensus logic and block propagation for Ethereum. Since The Merge, there has only been one, single Ethereum chain.
```

We could also remove some unnecessary words here if we wanted. Feel free to ignore if you disagree.

```suggestion
  The Beacon Chain shipped on December 1, 2020. At the time, it was a separate chain to Ethereum Mainnet. It ran proof-of-stake but did not handle any transactions. The Beacon Chain was merged with the original Ethereum Mainnet in 2022 to take over the consensus logic and block propagation for Ethereum. Since The Merge, there has only been one Ethereum chain.
```

</UpgradeStatus>

## What was the Beacon Chain? {#what-is-the-beacon-chain}

> The Beacon Chain was the name given to the original proof-of-stake blockchain that was launched in 2020. It was created to ensure the proof-of-stake consensus logic was sound and sustainable before enabling it on Ethereum Mainnet. Therefore, it ran alongisde the original proof-of-work Ethereum. Switching off proof-of-work and switching on proof-of-stake on Ethereum required instructing the Beacon Chain to accept transactions from the original Ethereum chain, bundle them into blocks and then oprganize the blocks into a block chain using a proof-of-stake based consensus mechanism. At the same moment, the original Ethereum clients turned off their mining, block propagation and consensus logic, handing that all over to the Beacon Chain. This event was known as [The Merge](/upgrades/merge/). Once The Merge happened, there were no longer two blockchains, there was just one single proof-of-stake Ethereum.

Reviewer: ```suggestion
The Beacon Chain was the name of the original proof-of-stake blockchain that was launched in 2020. It was created to ensure the proof-of-stake consensus logic was sound and sustainable before enabling it on Ethereum Mainnet. Therefore, it ran alongside the original proof-of-work Ethereum. Switching off proof-of-work and switching on proof-of-stake on Ethereum required instructing the Beacon Chain to accept transactions from the original Ethereum chain, bundle them into blocks and then organize them into a blockchain using a proof-of-stake based consensus mechanism. At the same moment, the original Ethereum clients turned off their mining, block propagation and consensus logic, handing that all over to the Beacon Chain. This event was known as [The Merge](/upgrades/merge/). Once The Merge happened, there were no longer two blockchains; there was just one proof-of-stake Ethereum chain.
```

Reviewer: Noticed we're saying 'proof-of-stake based' instead of 'proof-of-stake' in a lot of places now. Curious what the thinking behind this is?

Author: feedback from wider EF about definition of "consensus mechanism". The consensus mechanism is more than just the proof-of-stake part, it also includes the social coordination, fork choice rule, incentives and penalties, etc. It is hard to wrap all this into simple wordage, so we started using "pos-based mechanism" as the neatest compromise between readability and accuracy we could think of (we being Paul and I).

Reviewer: Makes sense to me, was just curious :-)

### Beacon Chain and The Merge {#merge-and-beacon-chain}

> The Beacon Chain, at first, existed separately to Ethereum Mainnet but they were merged together in 2022.

Reviewer: ```suggestion
At first, The Beacon Chain existed separately from Ethereum Mainnet, but they were merged in 2022.
```
